azurerm/modules/azurerm-kv/providers.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "azurerm" {
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
```

Comment:
Remove the provider from the module itself, they're usually defined on the root level and auto-cascade down

Reply:
thanks for pointing at it Jack, I have removed it now